gatsby-theme-mds/src/css/Leftnav.css

```diff
@@ -0,0 +1,5 @@
+.title {
```

Comment:
Please use helper padding classes and heading component with appearance, remove this class definition.